proto/osmosis/protorev/v1beta1/query.proto

```diff
@@ -108,6 +108,12 @@ service Query {
       returns (QueryGetProtoRevEnabledResponse) {
     option (google.api.http).get = "/osmosis/v14/protorev/enabled";
   }
+
+  // GetProtoRevPool queries whether the module is enabled or not
```

Comment:
Do we want this to also return the hot routes as well or only the highest liquidity pool?
```suggestion
  // GetProtoRevPool queries the pool id used for arbitrage route building given a pair of denominations
```

Reply:
If we only care about the highest liquidity pool id, I would update the doc to say that.